adele-ticket-master-api/src/main/java/com/microservicesteam/adele/ticketmaster/events/TicketsPaid.java

```diff
@@ -0,0 +1,7 @@
+package com.microservicesteam.adele.ticketmaster.events;
+
+import org.immutables.value.Value;
+
+@Value.Immutable
+public interface TicketsPaid extends TicketsEvent {
```

Comment:
As we discussed my intention was letting you have a look on the structure for early feedback.